### PR TITLE
Modified tez.sh to install Tez from repo.

### PR DESCRIPTION
**Pull request by jmikula**
Addresses https://github.com/GoogleCloudPlatform/dataproc-initialization-actions/issues/35

### CODE REVIEW

File: tez/tez.sh [hunk 1]
```diff
@@ -17,62 +17,19 @@ set -x -e
 #  Define important variables
 tez_version="0.7.0"
 protobuf_version="2.5.0"
```
Comment (pmkc) [inline tez/tez.sh]: I would take these out.

**Comment by pmkc** [comment]
LGTM

**Comment by bzz** [comment]
Do you guys think this is ready to be merged and #35 closed?

**Comment by evilsoapbox** [comment]
Since this has a LGTM from @pmkc I am going to go forth and merge/close.

**Pull request closed without merging**